CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
-## 1.0
+## 1.0.0
 
 Initial release.
+
+# 1.1.0
```

Comment:
Small thing here. The heading level should be `##` and the newer one should be at the top. And date it to the next scheduled release, please.

Reply:
Done